index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="kelwilson" />
+    <title>kelwilson-portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/boxicons@latest/css/boxicons.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="header-wrap">
+        <div class="top-background-img"></div>
+        <nav>
+           <i class="bx bx-menu"></i>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="portfolio-wrap">
+        <div class="portfolio-header">
+          <h1>
+            Kelly <br/>
+            Besong
+          </h1>
+
+          <p>
+            Hello! I am a software developer! I can 
+            help you build a product, feature or 
+            website. Take a look of my works. If you 
+            like what you see and have a project 
+            you need coded, don’t hesitate and 
+            contact me.
+          </p>
+        </div>
+        <div class="start-btn">
+          <button type="button">Start collaboration</button>
+        </div>
```

Comment:
- [ ] An _anchor tag_, `a`, is missing for the button. Kindly ensure to add an _anchor tag_ so as to enable link to be added to the button.

Reply:
**I tried wrapping an anchor tag around my button but it gave me some linters errors.**

-  here is a snapshot of the result I got after I added the tag

![code-re](https://user-images.githubusercontent.com/97304715/170225341-185d2384-e04c-4e29-8ffa-6cb7c6b6afc3.jpg)
 as its shown here on my code editor on the left hand side and the errors on my terminal on the right

Reply:
- You can try wrapping the `<button>` around the `<a>`